### PR TITLE
Adding webgl shape kernel

### PR DESCRIPTION
**Pull request by duli2012**
**Description**: Describe your changes.
Adding webgl shape kernel

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: js/web/lib/onnxjs/backends/webgl/ops/shape.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// Licensed under the MIT License.
+
+import {Shape} from '../../../ops/shape';
+import {Tensor} from '../../../tensor';
+import {WebGLInferenceHandler} from '../inference-handler';
+
+
+export class WebGLShape extends Shape {
+  run(inferenceHandler: WebGLInferenceHandler, inputs: Tensor[]): Tensor[] {
+    const shape = inputs[0].dims.slice(0);
```
Comment (fs-eire) [inline js/web/lib/onnxjs/backends/webgl/ops/shape.ts:11]: no need to create a copy

File: js/web/test/data/ops/shape.jsonc [hunk 2]
```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "name": "Reshape with '0' and '-1' in the shape tensor input",
```
Comment (fs-eire) [inline js/web/test/data/ops/shape.jsonc:3]: the name should be updated

File: js/web/lib/onnxjs/backends/webgl/ops/shape.ts (hunk 1 above)
Comment (duli2012) [reply js/web/lib/onnxjs/backends/webgl/ops/shape.ts]: fixed.

File: js/web/test/data/ops/shape.jsonc (hunk 2 above)
Comment (duli2012) [reply js/web/test/data/ops/shape.jsonc]: done.

**Review by fs-eire — APPROVED** [review]

**Pull request merged**